### ISSUE TITLE
UX Improvements

### ISSUE THREAD

**Issue by sepehr** [gabrielelana/mongodb-shell-extensions#4]
[MongoHacker](https://github.com/TylerBrock/mongo-hacker)-like UX improvements are very much appreciated. As integrating both extensions is a little bit tricky and we all prefer all-in-alls!

*P.S. Thank you for this great contribution.*

**Comment by gabrielelana** [comment]
@sepehr yep, unfortunately we are pretty much mutually exclusive :disappointed: 

What do you like to have in particular? What do you use?
